### ISSUE TITLE
Semrush widget endpoint no more working 

### ISSUE THREAD

**Issue by joeforjoomla** [eyecatchup/SEOstats#139]
Currently the SEMRush endpoint is returning  "ERROR" for all domains. The library is no more working.

http://widget.semrush.com/widget.php?action=report&type=organic&db=us&domain=www.cnn.com

**Comment by siddiquimeraj** [comment]
Yah it throws 404 on browser but when i query using  the library it gives me output

eg . output
[domain rank] => Array
        (
            [The SEMRush Rank (rating of sites by the number of visitors coming from the first 20 search results).] => 709139
            [Number of Keywords this site has in the TOP20 organic results.] => 2312
            [Estimated number of visitors coming from the first 20 search results (per month).] => 680
            [Estimated cost of purchasing the same number of visitors through Ads.] => 3115
            [F0] => 9307201
            [X0] => 6
            [D1] => 6
            [F1] => 55836732
            [X1] => 80
            [D2] => 80
            [X2] => 198
            [D3] => 2226
            [X3] => 251
            [X4] => 179
            [X5] => 250
            [X6] => 237
            [F7] => 17179869184
            [X7] => 278
            [X8] => 270
            [X9] => 294
            [Ca] => 21,6,13,5,1
            [Xa] => 269
            [Cb] => 279
            [Tb] => 440
            [Dc] => 39,18,13,8,7
            [Oe] => 60
            [Oi] => 87
            [Ol] => 35
            [On] => 37
            [Fl] => 1:10.89,2:1.42,5:0.60,6:8.52,31:0.04
            [Estimated number of competitors in organic search.] => 678
        )

All I am looking is can some one tell me what  the keys means in this array??? Except the long string keys. Of course any one knows that.  I tried semrush  to find the meaning of those keys but unsuccessful.

Thanks in advance

**Comment by joeforjoomla** [comment]
The SemRush widget endpoint seems definitely DEAD

**Comment by sam2332** [comment]
Indeed, also I believe you guys have the wrong email address as I know
nothing about SEOstats other than I've used the product in the past

On Dec 2, 2016 6:57 AM, "JoeforJoomla Boy" <notifications@github.com> wrote:

The SemRush widget endpoint seems definitely DEAD

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/eyecatchup/SEOstats/issues/139#issuecomment-264438758>,
or mute the thread
